src/pages/components/pagination/accessibility.mdx

```diff
@@ -45,11 +45,8 @@ other accessibility considerations, some of which are described below.
 #### Pagination
 
 The tab order goes from left to right through the controls in the pagination
-variant. On focus, the dropdowns are opened with `Space` or with up or down
-arrows, which also cycle through the values. Both `Space` and `Enter` select a
-value and close the dropdown. The dropdown can also be closed by pressing `Esc`.
-The previous and next page arrow buttons are activated by `Space` or `Enter`
-keys.
+variant. On focus, the selects are opened with `Space` or with
+'Up`or`Down`arrows, which also cycle through the values. Both`Space`and`Enter`select a value and close the select. The select can also be closed by pressing`Esc`. The previous and next page arrow buttons are activated by pressing `Space`or`Enter`.
```

Comment:
`Up`or `Down` arrows, which also cycle through the values. Both`Space` and `Enter` select a value and close the select. The select can also be closed by pressing `Esc`. The previous and next page arrow buttons are activated by pressing `Space` or `Enter`.